packages/css/src/components/form/form.stories.ts

```diff
@@ -0,0 +1,21 @@
+import { storiesOfForm } from '@dso-toolkit/sources';
+import { storiesOf } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+import { formGroupInputTemplate } from './form-groups/input/form-group-input.template';
+
+import { formTemplate } from './form.template';
+import readme from './readme.md';
+
+storiesOfForm(
+  {
+    module,
+    storiesOf,
+    readme
+  },
+  {
+    formTemplate,
+    formGroupDecorator: story => html`<form><legend>Example</legend><fieldset>${story()}</fieldset></legend></form>`,
```

Comment:
Hier moeten we iets mee, want dit is ongeldige HTML en schaalt niet. Morgen ff verder brainen